### DISCUSSION TITLE
Use new TLS1.3 feature as default on all configurations استفاده از تنظیمات TLS1.3 بر روی همه‌ی کانشن ها و سابسکریپشن‌ ها

### DISCUSSION THREAD

**Discussion by raminrez** [hiddify/Hiddify-Manager#177] [Ideas ایدهها]
درود دوستان
طبق توضیحات [این توویت](https://twitter.com/vahidfarid/status/1623971797979787265) ایا امکانش هست که TLS1.3 به‌صورت پیشفرض روی همه‌ کانشن ها تنظیم بشه؟

**Comment by hiddify-com** [comment]
در حال حاضر خودکار هست
از اونجایی که 90 درصد سایت ها tls 1.2 هستند ما اگه tls1.2 را ببندیم تابلو میشه که سایت فیلترشکنه
از طرفی اون توییت اغراق شده برای ESNI نیاز به تنظیمات dns هست که فیلترچیان بسته اند اونو
